packages/capacitor/src/hooks/useEdgeInsets.ts

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react'
+
+import { useOrientation } from './useOrientation'
+
+type EdgeInsets = {
+  top: number
+  right: number
+  bottom: number
+  left: number
+}
+
+function getPropertyValue(key: string): number {
+  const propertyValue = getComputedStyle(
+    document.documentElement
+  ).getPropertyValue(key)
+
+  return Number(propertyValue.replace('px', ''))
+}
+
+function useEdgeInsets(): EdgeInsets {
+  const orientation = useOrientation()
+  const [edgeInsets, setEdgeInsets] = useState<EdgeInsets>({
+    top: 0,
+    right: 0,
+    bottom: 0,
+    left: 0,
+  })
+
+  useEffect(() => {
+    setEdgeInsets({
+      top: getPropertyValue('--safe-area-inset-top'),
```

Comment:
REQ: This requires the SCSS for settings the CSS variables.

Reply:
How would you include these in the package?
I'm not clear on what you mean